### ISSUE TITLE
logs_rubi_val_oe.json not getting generated

### ISSUE THREAD

**Issue by abhipsabasu** [cdancette/rubi.bootstrap.pytorch#15]
When I run rubi, the following files are getting generated. No logs_rubi_val_oe.json is getting generated. Is it the rubi accuracy that is present in logs_val_oe.json? The numbers seem to be tallying with that of rubi as reported in the paper. Attaching the screenshot below.

![image](https://user-images.githubusercontent.com/10346347/143234093-66eb08b6-0bde-429c-9b07-e516b0b9ad41.png)

**Comment by cdancette** [comment]
Can you check the _rubi_val_oe.json as well ? It seems more likely that logs are there, I'm not sure why it's missing the `log` prefix

**Comment by abhipsabasu** [comment]
On checking _rubi_val_oe.json, I saw the overall accuracy is max 37.22, whereas in logs_val_oe.json, the max accuracy is 48.01. So the latter score is matching with that of the paper isnt it?

**Comment by cdancette** [comment]
Yes you are correct
 In _rubi_val_oe there are the scores obtained by keeping the question only
branch.


Le jeu. 25 nov. 2021, 05:05, abhipsabasu ***@***.***> a
écrit :

**Comment by abhipsabasu** [comment]
Okay thanks a lot

**Comment by abhipsabasu** [comment]
Hi,
In the paper, did you report the highest validation accuracy? or the validation accuracy obtained at the end of training?
